tests/test_get_all_sentences.rs

```diff
@@ -1,46 +1,42 @@
 extern crate reqwest;
 extern crate uuid;
+extern crate postgres;
 
 #[macro_use] extern crate serde_derive;
 
 use reqwest::StatusCode;
 
+use postgres::Connection;
+
 mod db;
 
+use db::DatabaseHandler;
+
 #[path = "../utils/tests_commons.rs"]
 mod tests_commons;
 
 #[test]
 fn test_get_all_sentences_returns_200() {
 
-    let connection = db::get_connection();
-    db::clear(&connection);
+    let connection: Connection = DatabaseHandler::connect_and_clean();
 
     let first_english_iso639_3 = "eng";
-    db::insert_language(
-        &connection,
-        &first_english_iso639_3,
-    );
+    connection.insert_language(&first_english_iso639_3);
 
     let second_english_iso639_3 = "fra";
-    db::insert_language(
-        &connection,
-        &second_english_iso639_3,
-    );
+    connection.insert_language(&second_english_iso639_3);
 
     let first_english_uuid = uuid::Uuid::new_v4();
     let first_english_text = "This is one sentence";
-    db::insert_sentence(
-        &connection,
+    connection.insert_sentence(
```

Comment:
i think we should get a "insert_sentence" version without the need for uuid , that autogenerate the uuid (using postgres) and that return the uuid , as most of the time the uuid is not used elsewhere in the test 

so that we could use 

```
connection.insert_sentence(first_text, first_iso639_3);
```
on one line which would allow for the test to fit in one screen without need to scroll nor impairing readability

Reply:
and we could hve a "insert_sentence_with_uuid" version that take the uuid as parameters 

and/or one that take directly a "Sentence" object

Reply:
Got it, what about doing that directly into the next PR ?

Reply:
ok